article/sections/_7_latent_confounding.tex

```diff
@@ -0,0 +1,209 @@
+confounders, the assumptions for which variables are confounded, and the models 
+needed to estimate the causal effects. 
+We do that using a simplified simulation scenario to investigate the usefulness 
+and pitfalls of the deconfounder approach for generating accurate model estimates. 
+
+
+\subsection{Examples of confounding}
+\label{sec:confounding-examples}
+
+Confounding occurs when a certain (confounding) variable induces variations in
+both the outcome as well as the treatment (policy) variables of interest, 
+creating correlation between the treatment and the outcome that is not caused
+by the treatment variable itself. When the confounding variable (or variables) is 
+observed, we can control for the confounding effect, and there exists many 
+methods in the literature for how to do that, including post-stratification,
+multiple regression, propensity score methods, etc. It is when the 
+confounding variable is unobserved that the problem becomes significantly 
+more challenging. 
+For an illustration, imagine we're interested in the effect of adding a bike on the mode share of bicycling in a given neighborhood. 
+To estimate this effect, one strategy that a modeler may use is to develop a disaggregate mode choice model with a dummy variable for whether a bikelane exists between an individual's trip origin and destination, and then treating the coefficient on this variable as the causal effect of adding a bikelane on the log-odds of choosing to bike. 
+The problem with such strategy is that individuals may self-select to live in an area where bicycle infrastructure exists because they have a preference for commuting by bike. 
+In other words, if we define an additional variable to encode a person's latent inherent preference for biking, then this variable determines both a person's likelihood for living in an area with an existing bicycle infrastructure, as well as whether that person choose to bike. 
+This variable is subsequently expected not to lack "balance" across the treatment and control groups; those who live near a bicycle lane are expected to have higher preference for biking than those who don't. 
+Without accounting for this latent confounder, we risk getting very biased estimates 
+of the effects of our treatment variable of interest, created by the 
+induced variations by the confounder in both the treatment and the outcome 
+variable. 
+
+
+
+The problem of latent confounding and ommitted variable bias is widely 
+acknowledged in the transportation literature, and demand modelers can draw on 
+a list of method to account for confouding in some specific circumstances. 
+Integrated choice and latent variable (ICLV) models are a way to account for 
+the effect of unobserved attitudinal variables which may affect the selection 
+of some individuals into some treatment level, as well as an outcome of 
+interest. For example, a person's unobserved beliefs and attitudes towards 
```

Comment:
@hassan-obeid for visibility.

I don't know how I never caught this before.
This statement is incorrect.
See the image of ICLV models in section 3, Figure 14, or for an online reference, see [here](https://www.researchgate.net/figure/The-ICLV-model-framework-adapted-from-Ben-Akiva-et-al-2002_fig1_302922031).
ICLV models are about latent mediators, not latent confounders.

Reply:
Note that it's tricky because these latent mediators may confound other explanatory variables.
E.g. attitudes are a mediator between sociodemographics and utility.
Attitudes may also be a confounder of level-of-service (via residential location) and utility.